docs/tutorials/wrfv3/spack-wrfv3.md

```diff
@@ -0,0 +1,296 @@
+# HPC Toolkit - Install and Run Weather Research and Forecasting (WRF) Model on a Slurm Cluster
+
+HPC Toolkit is an open-source software offered by Google Cloud which makes it
+easy for customers to deploy HPC environments on Google Cloud.
+
+In this tutorial you will use the HPC Toolkit to:
+
+* Deploy a [Slurm](https://github.com/SchedMD/slurm-gcp#readme) HPC cluster on
+  Google Cloud
+* Use [Spack](https://spack.io/) to install the Weather Research and Forecasting (WRF) Model application and all of
+  its dependencies
+* Run a [Weather Research and Forecasting (WRF) Model](https://www.mmm.ucar.edu/weather-research-and-forecasting-model) job on your newly provisioned
+  cluster
+* Tear down the cluster
+
```

Comment:
Here, I would maybe add the following lines at the bottom.

"Deploying the cluster and installing the applications dependencies may take 1.5 hours if no spack cache is provided. With a complete spack cache, this time can be reduce to 6 minutes." (please check this time).

Reply:
Done. PTAL as final was slightly different that what you suggested.